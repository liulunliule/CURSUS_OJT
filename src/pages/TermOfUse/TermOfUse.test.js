import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom/extend-expect';
import { MemoryRouter } from 'react-router-dom'; // Import MemoryRouter
import TermOfUse from '.';

test('test TermOfUse Page', () => {
  const tree = renderer.create(
      <MemoryRouter>
        <TermOfUse/>
      </MemoryRouter>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
