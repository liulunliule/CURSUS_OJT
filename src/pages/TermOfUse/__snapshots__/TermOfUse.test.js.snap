// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test TermOfUse Page 1`] = `
<div>
  <div
    className="policy"
  >
    <div
      className="policy-container"
    >
      <div
        className="policy__inner"
      >
        <div
          className="policy__toolbar"
        >
          <div
            className="policy__toolbar-home"
          >
            <a
              href="/"
              onClick={[Function]}
            >
              Home 
            </a>
            / Policy privacy
          </div>
          <div
            className="policy__toolbar-search"
          >
            <div
              className="policy__toolbar-icon"
            >
              <span
                aria-label="search"
                className="anticon anticon-search"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  data-icon="search"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M909.6 854.5L649.9 594.8C690.2 542.7 712 479 712 412c0-80.2-31.3-155.4-87.9-212.1-56.6-56.7-132-87.9-212.1-87.9s-155.5 31.3-212.1 87.9C143.2 256.5 112 331.8 112 412c0 80.1 31.3 155.5 87.9 212.1C256.5 680.8 331.8 712 412 712c67 0 130.6-21.8 182.7-62l259.7 259.6a8.2 8.2 0 0011.6 0l43.6-43.5a8.2 8.2 0 000-11.6zM570.4 570.4C528 612.7 471.8 636 412 636s-116-23.3-158.4-65.6C211.3 528 188 471.8 188 412s23.3-116.1 65.6-158.4C296 211.3 352.2 188 412 188s116.1 23.2 158.4 65.6S636 352.2 636 412s-23.3 116.1-65.6 158.4z"
                  />
                </svg>
              </span>
            </div>
            <input
              className="policy__toolbar-input"
              id=""
              name=""
              placeholder="Search"
              type="text"
            />
          </div>
        </div>
        <h1
          className="policy__title"
        >
          Policy privacy
        </h1>
      </div>
    </div>
    <div
      className="policy__main"
    >
      <div
        className="policy-container"
      >
        <div
          className="policy__catalogue"
        >
          <ul>
            <li
              className="policy__catalogue-item"
            >
              <a
                className=""
                href="/secondLayout/policy"
                onClick={[Function]}
              >
                Term of Use
              </a>
            </li>
            <li
              className="policy__catalogue-item"
            >
              <a
                className=""
                href="/secondLayout/term"
                onClick={[Function]}
              >
                Privacy Policy
              </a>
            </li>
          </ul>
        </div>
        <div
          className="policy__content"
        >
          <p
            className="policy__content-title"
          >
            <i>
              Last Updated:
              <strong>
                August 1, 2020
              </strong>
            </i>
          </p>
          <p
            className="policy__content-desc"
          >
            Welcome to Cursus. Your privacy is important to us. This Privacy Policy explains how we collect, use, and protect your personal information when you access or use our website.
          </p>
          <p
            className="policy__content-title"
          >
            <b>
              1. Information We Collect
            </b>
          </p>
          <p
            className="policy__content-desc"
          >
            We collect personal information that you voluntarily provide to us when you register for an account, use our services, or contact us. This may include: Your name Email address Phone number Payment information Any other information you choose to provide
          </p>
          <p
            className="policy__content-desc"
          >
            We may also automatically collect certain information when you visit our website, including: IP address Browser type Operating system Pages you view The time and date of your visit The referring URL
          </p>
          <p
            className="policy__content-title"
          >
            <b>
              2. How We Use Your Information
            </b>
          </p>
          <p
            className="policy__content-desc"
          >
            All content on our website, including text, graphics, logos, and images, is the property of [Your Website Name] or its content suppliers and is protected by international copyright laws. You agree not to use any of our intellectual property without our prior written consent.
          </p>
          <p
            className="policy__content-desc"
          >
            We reserve the right to terminate or suspend your account and access to our services at our sole discretion, without notice, for conduct that we believe violates these Terms or is harmful to other users of our website.
          </p>
          <p
            className="policy__content-desc"
          >
            Our services are provided on an "as is" and "as available" basis. We make no warranties, express or implied, regarding the operation of our website or the information, content, or materials included therein.
          </p>
          <p
            className="policy__content-desc"
          >
            To the fullest extent permitted by law, [Your Website Name] shall not be liable for any indirect, incidental, special, or consequential damages arising out of or in connection with the use of our services.
          </p>
          <p
            className="policy__content-desc"
          >
            We reserve the right to modify these Terms at any time. We will notify you of any changes by posting the new Terms on our website. Your continued use of our services after such changes constitutes your acceptance of the new Terms.
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
